README.md

```diff
@@ -71,6 +71,22 @@ kube-system          kube-scheduler-dra-example-driver-cluster-control-plane
 local-path-storage   local-path-provisioner-7dbf974f64-9jmc7                            1/1     Running   0          1m
 ```
 
+When the validating admission webhook is enabled (as it is by default), cert-manager and its CRDs must be installed.
```

Comment:
Instead of taking on a cert-manager dependency, I experimented with Helm's builtin in [`genSignedCert`](https://helm.sh/docs/chart_template_guide/function_list/#gensignedcert) and adjacent template functions. The main functional caveat with those is that every `helm upgrade` will generate a new cert, but if the Deployment doesn't change, then a new Pod will not be created to pick up to that new cert, leading to connection issues until a new Pod is created. That might be acceptable for what this example driver will be used for, but probably isn't a pattern we want to establish for "real" drivers.

Reply:
Webhook doesn't seem to be vital for a DRA driver to function. Why not keep the webhook disabled in values, and instruct to _enable_ the webhook if needed, given that it has a dependency and that dependency is not part of the chart?

Reply:
Can do. Disabled it by default and reworded this to match.